cmake.config/CMakeLists.txt

```diff
@@ -186,3 +186,15 @@ configure_file (
   "${PROJECT_SOURCE_DIR}/cmake.config/pathdef.c.in"
   "${PROJECT_BINARY_DIR}/cmake.config/auto/pathdef.c"
   ESCAPE_QUOTES)
+
+check_c_source_compiles("
+#include <intrin.h>
+
+int main(void)
+{
+  unsigned long index;
+  unsigned char mask = 0x8000;
+  _BitScanForward64(&index, mask);
+  return 0;
+}
+" HAVE_BITSCANFORWARD64)
```

Comment:
Shouldn't there also be a change to `cmake.config/config.h.in`?

Reply:
And I think this should be place near other `check_c_source_compiles` checks instead of being at the end of the file.